terraform/staging/feature_flags.tf

```diff
@@ -6,3 +6,12 @@ resource "aws_secretsmanager_secret_version" "show_historic_data_feature_flag" {
   secret_id     = aws_secretsmanager_secret.show_historic_data_feature_flag.id
   secret_string = "true"
 }
+
+resource "aws_secretsmanager_secret" "deallocation_date_feature_flag" {
+    name = "social_care_case_viewer_api_deallocation_date"
+}
+
+resource "aws_secretsmanager_secret_version" "social_care_case_viewer_api_deallocation_date" {
+    secret_id     = aws_secretsmanager_secret.deallocation_date_feature_flag.id
+    secret_string = "true"
+}
```

Comment:
If we want to use these values as environment variables, we'll need to update the serverless config for `SOCIAL_CARE_DEALLOCATION_DATE` like what we've done for historic data:

https://github.com/LBHackney-IT/social-care-case-viewer-api/blob/427aa8b04dcdd99fa92b4718eb7089dadb482f80/serverless.yml#L38